plugins/Vestige/Vestige.cpp

```diff
@@ -363,7 +365,7 @@ void VestigeInstrument::loadFile( const QString & _file )
 	}
 
 	m_pluginMutex.lock();
-	m_plugin = new VstInstrumentPlugin( m_pluginDLL );
+	m_plugin = new VstInstrumentPlugin{m_pluginDLL, this};
```

Comment:
It's passing the instrument's `Model` to the pin connector so that the pin connector's `BoolModel`s have the instrument as their parent. I do the same for VST effects as well.

Reply:
`VestigeInstrument` (and all other plugins) get the parent model from the `lmms_plugin_main` function. In the constructor, it passes that pointer all the way down to `Plugin` which inherits from `Model`, and then `Model` sets it as its parent model.